### PR TITLE
UI tests: fix layout for Chrome > 62

### PR DESCRIPTION
**Pull request by breville**

### CODE REVIEW

File: dashboard/test/ui/features/blocklayout.feature [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Feature: Block auto-layout
+@chrome_after_62
```
Comment (joshlory) [inline dashboard/test/ui/features/blocklayout.feature]: This may need to be the first line.

**Comment by joshlory** [comment]
This will cause `blocklayout.feature` to ONLY run in Chrome 63, no other browsers?  (Safari, Firefox, Edge, ...)

**Comment by joshlory** [comment]
Actually never mind, as written this will also run in other browsers.  Maybe update the tag name?

```rb
if browser['browserName'] != 'chrome' || ...
```

**Comment by breville** [comment]
Deprecating in favour of https://github.com/code-dot-org/code-dot-org/pull/19654

**Pull request closed without merging**